tox.ini

```diff
@@ -19,27 +19,23 @@ whitelist_externals =
 # We removed cleanup from the end because it complicate Travis
 # debugging.
 commands =
-    molecule lint -s default
-    molecule converge -s default # calls: dependency, create and prepare
-    molecule syntax -s default
-    {env:TRAVIS_WAIT:} molecule idempotence -s default
-    molecule side-effect -s default
-    molecule verify -s default
-    molecule destroy -s default
-    molecule lint -s source
-    molecule converge -s source # calls: dependency, create and prepare
-    molecule syntax -s source
-    {env:TRAVIS_WAIT:} molecule idempotence -s source
-    molecule side-effect -s source
-    molecule verify -s source
-    molecule destroy -s source
-    molecule lint -s dynamic
-    molecule converge -s dynamic # calls: dependency, create and prepare
-    molecule syntax -s dynamic
-    {env:TRAVIS_WAIT:} molecule idempotence -s dynamic
-    molecule side-effect -s dynamic
-    molecule verify -s dynamic
-    # only the last scenario should lack a "destroy" task. (It wastes time, and makes debugging more difficult.)
+    {env:TRAVIS_WAIT:} molecule test -s default
+    {env:TRAVIS_WAIT:} molecule test -s source
```

Comment:
Will the cronjobs (to run all tests on all python versions) run the dynamic tests and upgrade tests as well?

Reply:
currently yes, they will, and I think it is too much (9 jobs).

Reply:
I thought (but am not certain) we agreed in the meeting that since it runs overnight, we would run all 9 jobs.

The total length may be too long, but we can reduce it later.